find-datalad-repos.py

```diff
@@ -283,9 +283,13 @@ def get_new_collection(self, searcher: "GHDataladSearcher") -> List[DataladRepo]
         return collection
 
     def get_report(self) -> str:
+        news = (
+            "{self.new_repos} new datasets",
+            "{self.new_runs} new `datalad run` users",
+        )
         return (
-            f"Added {self.new_hits} new hits: {self.new_repos} new datasets"
-            f" and {self.new_runs} new `datalad run` users"
+            f"Added {self.new_hits} new hits: " +
```

Comment:
Are you sure you want to end with a ": " if there are no hits at all?  (There could still be things to commit if, say, a repo gained or lost stars.)

Reply:
Good point! Will tune it up. Thank you!

Reply:
done